cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,12 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
```

Comment:
The test case does not contain any mocks or stubs to isolate the test scenario. Integrating with live services can result in unstable tests. Consider mocking the network requests to isolate the test scenario.